### PR TITLE
Changing version_added on resources to appease Ansible

### PR DESCRIPTION
**Pull request by rambleraptor**
Changing version_added on resources to appease Ansible

Because of the rebase, there should be resources that Ansible doesn't necessarily like.

These linters are a pain because I'm not entirely convinced that the Magician is running everything the same as Shippable is.

-----------------------------------------------------------------
# [all]
Changing version_added on resources to appease Ansible
## [ansible]
Changing version_added on some resources

### CODE REVIEW

**Comment by danawillow** [comment]
It looks like there's a bunch of unrelated changes in here, mind fixing that before I review?

**Comment by rambleraptor** [comment]
Woah, yeah. I'll definitely do that.

**Comment by rambleraptor** [comment]
@ndmckinley any ideas what I'm doing wrong on this branch? I'm not entirely sure how to force it to merge into the `rebase` branch

File: .ci/magic-modules/create-pr.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ if [ "$BRANCH_NAME" = "$ORIGINAL_PR_BRANCH" ]; then
     fi
 
     git checkout -b "$BRANCH_NAME"
-    if ANSIBLE_PR=$(hub pull-request -b "$ANSIBLE_REPO_USER/ansible:devel" -F ./downstream_body); then
+    if ANSIBLE_PR=$(hub pull-request -b "$ANSIBLE_REPO_USER/ansible:rebase" -F ./downstream_body); then
```
Comment (ndmckinley) [inline .ci/magic-modules/create-pr.sh]: I feel suspicious about this.  Is this the plan as discussed?  Can you elaborate on why this is necessary?

Comment (rambleraptor) [reply .ci/magic-modules/create-pr.sh]: I was trying to create a PR that was heading to the `rebase` branch.

For whatever reason, the PR for Ansible I have right now looks how I want it to. It adds a whole bunch of commits to devel, but keeps the commit history in tact. There's a merge commit (I made that commit myself) and then the difference between the merge commit + HEAD is the difference I would expect from this PR.

Comment (rambleraptor) [reply .ci/magic-modules/create-pr.sh]: Truth be told, I haven't thought about this in quite a while and I'm not entirely sure what the plan we agreed upon was.

Comment (ndmckinley) [reply .ci/magic-modules/create-pr.sh]: Undo this stuff and you're good!

**Pull request merged**